pcdsdevices/pim.py

```diff
@@ -559,17 +559,48 @@ class IM2K0(LCLS2ImagerBase):
     # Nothing else! No power meter, no zoom/focus, no filter wheel...
 
 
-class IM3L0(PPM):
+class K2700(BaseInterface, Device):
+    """
+    Keithley 2700 digital multimeter.
+
+    Currently supports reading voltage and current, direct and
+    alternating, but can be extended to measure other quantities
+    (resistance, temperature), have configurable range and integration
+    time, and allow for remote control of a K2700.
+    """
+    idn = Cpt(EpicsSignalRO, ":Identity", kind="normal")
+    reading = Cpt(EpicsSignalRO, ":Reading", kind="normal")
+    dcv = Cpt(EpicsSignalRO, ":GetDCV", kind="normal")
+    acv = Cpt(EpicsSignalRO, ":GetACV", kind="normal")
+    dci = Cpt(EpicsSignalRO, ":GetDCI", kind="normal")
+    aci = Cpt(EpicsSignalRO, ":GetACI", kind="normal")
+
+
+class IM3L0_K2700(K2700):
     """
-    One-off subclass of PPM to add Keithley readout to this device's detailed screen.
+    One-off subclass of K2700 to use a pydm screen specific to this device.
 
-    Identical to PPM class, but has an altered detailed screen that adds an embedded
-    pydm display that allows acces to the Keithley monitoring this device; subclass
-    allows for adding Keithley readout to only this device instead of all PPM instances.
+    Identical to K2700 class, but uses a pydm screen for this particular device
+    in place of the default detailed screen. To be used in conjunction with
+    IM3L0 as this Keithley is added to that imager for detailed power readouts.
     """
     pass
 
 
+@reorder_components(
+    end_with=['k2700', 'power_meter', 'yag_thermocouple', 'led']
+)
+class IM3L0(PPM):
+    """
+    One-off subclass of PPM to include this device's Keithely 2700
+
+    Includes a Keithley 2700 digital multimeter on top of the PPM class, mainly
+    so it shows up on this device's detailed screen.
+    """
+    k2700 = Cpt(IM3L0_K2700, ':SPM:K2700', kind='normal',
```

Comment:
Sub-devices don't need "kinds", just so you're aware. It doesn't technically hurt anything to have this here, but it also has the same effect as leaving it off.

`kind` is more of a "leaf Signal" sort of thing (as it tells the data acquisition routines what's important for plotting and such) ([ref](https://confluence.slac.stanford.edu/display/PCDS/How+to+Create+an+Ophyd+Device#HowtoCreateanOphydDevice-Choosingtheright%22kind%22))
```suggestion
    k2700 = Cpt(IM3L0_K2700, ':SPM:K2700',
```